src/main/java/de/danielbechler/diff/node/DiffNode.java

```diff
@@ -67,6 +67,7 @@
 	private Class<?> valueType;
 	private TypeInfo valueTypeInfo;
 	private IdentityStrategy childIdentityStrategy;
+	private Set<String> categoriesFromConfig;
```

Comment:
The name is a bit too specific for my taste. I would prefer `additionalCategories` because it makes the `Set` more versatile.